dockers/docker-sonic-restapi/critical_processes

```diff
@@ -1 +1,2 @@
 restapi
+cert_monitor
```

Comment:
is cert_monitor or cert_monitor.py?

Reply:
This needs to match the program name as defined in the supervisor.conf file, which in this case is `[program: cert_monitor]`, so this is correct.